.github/CONTRIBUTING.md

```diff
@@ -63,6 +63,7 @@ You can achieve this by running the following commands:
 gh repo fork keras-team/keras-cv --clone --remote
 cd keras-cv
 pip install ".[tests]"
+git config core.hooksPath .githooks
```

Comment:
Can we make this opt-in and a separate step.

Reply:
I did but this undermines our purpose of reducing all the time we already spend with formatting requests.

At least can we, in a subsequent PR, avoid to run all the jobs if the linting one is failing? 
It is really a waste of resources, including environmental ones

Reply:
> At least can we, in a subsequent PR, avoid to run all the jobs if the linting one is failing?
It is really a waste of resources, including environmental ones

It could be quite simple if we will use the `needs` syntax in Github Actions:
https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#jobsjob_idneeds

Reply:
I've added a new PR https://github.com/keras-team/keras-cv/pull/464